Makefile

```diff
@@ -172,28 +172,37 @@ clean-pnpm:
 	@cd $(THEME_GENERATOR_DIR) && $(PNPM) prune
 	@cd $(ICONS_DIR) && $(PNPM) prune
 	@cd $(DESKTOP_MODELS_DIR) && $(PNPM) prune
```

Comment:
I would also add some output here showing what is currently being executed, so it’s immediately clear what is being cleaned up where